eventbridge-pipes-sqs-to-eventbus-terraform/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+provider "aws" {
```

Comment:
Either explain to the user in the instructions how to deploy to a different region or remove this block

Reply:
Removed the region/profile block